tips/TIP-0024/tip-0024.md

```diff
@@ -0,0 +1,183 @@
+          </tr>
+        </table>
+      </details>
+  <tr>
+    <td>Nonce</td>
+    <td>uint64</td>
+    <td>The nonce which lets this message fulfill the PoW requirement.</td>
+  </tr>
+</table>
+
+## Syntactical validation
+
+The following criteria defines whether the message passes the syntactical validation:
+
+- The total length of the serialized message must not exceed `Max Message Length`.
+- `Network ID` must match the first 8 bytes of the BLAKE2b-256 hash of the `Network String` parameter.
+- Parents:
+  - `Parents Count` must be at least 1 and not larger than `Max Parents Count`.
+  - `Parents` must be sorted in lexicographical order.
+  - Each `Message ID` must be unique.
+- Payload (if present):
+  - `Payload Type` must match one of the values described under [Payloads](#payloads).
+  - `Data fields` must be correctly parsable in the context of the `Payload Type`.
+  - The payload itself must pass syntactic validation.
+- `Nonce` must be a valid solution of the message PoW as described in [TIP-12](../TIP-0012/tip-0012.md). The resulting PoW score must be larger or equal `Min PoW Score`.
+- There must be no trailing bytes after all message fields have been parsed.
+
+## Payloads
+
+While messages without a payload, i.e. `Payload Length` set to zero, are valid, such messages do not contain any information. As such, messages usually contain a payload. The detailed specification of each payload type is out of scope of this RFC. The following table lists all currently specified payloads that can be part of a message and links to their specification:
+
+| Payload Name | Type Value | TIP                                     |
+| ------------ | ---------- | --------------------------------------- |
+| Transaction  | 0          | [TIP-7](../TIP-0007/tip-0007.md)        |
+| Milestone    | 1          | [TIP-8](../TIP-0008/tip-0008.md)        |
+| Tagged Data  | 5          | [draft TIP-23](../TIP-0023/tip-0023.md) |
+
+### Tagged data payload
+
+This payload allows the addition of arbitrary data to the encapsulationg messages. An optional tag can be used to categorize the data. It is important to note that the tag is not considered by the protocol, it serves just as a marker for second layer applications.
+
+| Name         | Type                  | Description                                          |
+| ------------ | --------------------- | ---------------------------------------------------- |
+| Payload Type | uint32                | Set to *value 5* to denote an _Tagged Data Payload_. |
+| Tag Length   | uint8                 | The length of the following tag field in bytes.      |
+| Tag          | ByteArray[Tag Length] | The tag of the data.                                 |
+| Data         | ByteArray             | Binary data.                                         |
+
+*Syntactic validation*: `Tag Length` must not be larger than `Max Tag Length`. When `Tag Length` is 0, the `Tag` has length 0 and is omitted.
```

Comment:
Since you reference this payload in the table above, I don't think that this section is needed here.

Reply:
copy-paste mistake :see_no_evil: